ext/nnstreamer/tensor_filter/tensor_filter_lua.cc

```diff
@@ -70,17 +74,97 @@ void _fini_filter_lua (void) __attribute__((destructor));
 }
 #endif /* __cplusplus */
 
+/** @brief For getting value in Lua */
+static int
+tensor_index (lua_State *L)
+{
+  uint8_t** parray = (uint8_t **) luaL_checkudata (L, 1, "tensor");
+  int index = luaL_checkint (L, 2);
+  lua_pushnumber (L, (*parray)[index - 1]);
+
+  return 1;
```

Comment:
Q. Is there any meaning of this return value (e.g., 0 or 1)?

Reply:
It means the number of values returning to lua.
With this C function, the lua script `some_tensor[index]` gets single value.